OMGVA PoS/Business layer/Controllers/TaxController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using OMGVA_PoS.Data_layer.DTOs;
+using OMGVA_PoS.Data_layer.Mappers;
+using OMGVA_PoS.Data_layer.Repositories.Tax;
+
+namespace OMGVA_PoS.Business_layer.Controllers
+{
+    [ApiController]
+    [Route("tax")]
+    public class TaxController(ITaxRepository taxRepository) : ControllerBase
```

Comment:
I would recommend switching to `Controller` from `ControllerBase`. The `Controller` class extends `ControllerBase` and has more functionality for action filters and JSON helpers. Also, this would align better with the rest of the controllers.

Reply:
Used ControllerBase since [documentation](https://learn.microsoft.com/en-us/aspnet/core/web-api/?view=aspnetcore-9.0) mentioned that Controller is more for Web _Pages_, rather than just Web _APIs_. Since this controller is only for CRUD operations and we are not planning to use it for Views will leave it as is
![image](https://github.com/user-attachments/assets/34441e5f-075a-4954-92d8-ab613521bbe9)

Reply:
Alright, we might need to change this in the rest of the controllers then!